tech_grants_pipeline_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+export PYTHONPATH='.'
+# Creating training data
+python nutrition_labels/create_training_data.py --config_path configs/training_data/2021.03.08.test.ini &
```

Comment:
what is tests? is it evaluation? or some form of tests? if the latter then do we need this? we can just have tests

Reply:
yeh, it's not really a unit test or an evaluation. The reason I have it is because the main pipeline takes about 5 hours, so this was handy for a quick checkover. Arguably the unit tests should pick up on bugs, but perhaps there are other pipeline related checks, e.g. that the output of step1 works as the input of step2.

Also before running the 5 hour pipeline and having an issue right at the end, it might be good to check the test pipeline works first.